detox/scripts/postinstall.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh -e
```

Comment:
Why was this changed to `sh` from `bash`? Not saying  bash is absolutely needed here, but I don't think there is a linux installation that comes without bash?

Reply:
Alpine Linux doesn't have bash by default (it is still installable), that's why it launch scripts by sh. We use it because its image size.

Reply:
postinstall now uses node script to support Windows among Linux and macOS.
Give it a try (as soon as a new release is published), if you still have an issue, please report/submit a new PR